### PR TITLE
DEMO-47

### PR DESCRIPTION
**Pull request by sseerrnn**

### CODE REVIEW

File: src/user/users.controller.ts [hunk 1]
```diff
@@ -16,4 +27,41 @@ export class UsersController {
...
+    return this.usersService.findAll();
+  }
+
+  @Get(':id')
+  @HttpCode(HttpStatus.OK)
+  async getUser(@Param() params) {
+    return this.usersService.findById(params.id);
+  }
+
+  @Post()
```
Comment (karnjj) [inline src/user/users.controller.ts:42]: I think you should ignore `user.passwaord` before return data to client?

**Review by karnjj — COMMENTED** [review]
So, we use MongoDB to store users right?

File: src/user/users.controller.ts (hunk 1 above)
Comment (sseerrnn) [reply src/user/users.controller.ts:42]: Ok ,I will fix this

File: src/users/users.controller.ts [hunk 2]
```diff
@@ -1,13 +1,73 @@
...
+import { HttpStatus } from '@nestjs/common/enums';
+
+import { CreateUserDto } from './dto/create-user.dto';
 import { UsersService } from './users.service';
 
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
+  @Post('mockCreateUser')
```
Comment (pol-alones) [inline src/users/users.controller.ts:19]: This part of the code must not be in here. I think you can delete this and use unit test instead.

**Review by pol-alones — CHANGES_REQUESTED** [review]

File: src/users/users.controller.ts (hunk 2 above)
Comment (pol-alones) [reply src/users/users.controller.ts:19]: I think using integration test might be more appropriate in this case. The reason is that you can test database too.

**Pull request merged**